webpack.test.config.js

```diff
@@ -20,20 +20,23 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
+/* eslint-disable node/no-extraneous-require */
```

Comment:
Do you still need that webpack config file? :)

Reply:
Hmmm... No, you're right :)

I deleted this file :)